src/vm/arm/asmhelpers.S

```diff
@@ -287,7 +287,10 @@ LOCAL_LABEL(LNullThis):
 //
         NESTED_ENTRY TheUMEntryPrestub,_TEXT,NoHandler
 
-        push {r0-r4,lr}
+        .save {r0-r4,r7,r8,lr}
```

Comment:
Can we use PROLOG_PUSH followed by PROLOG_STACK_SAVE_OFFSET here and at several other places below? We have a convention to not to use the unwinder pseudoinstructions directly in the .S files.